### DISCUSSION TITLE
Serienreifes PCB?

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#629] [General]
Hallo Leute,

Gibt es jetzt eigentlich ein serienreifes PCB welches man erwerben kann?

Mein Nachbar und ich planen nun endlich jeweils ein BkW

Folgendes sollte klappen:

Victron VE.Direct Support für 100/20 MPPT und Smartshunt
Hoymiles HMS-600-2T (CMT2300A)

Unterstützung einer Pylontech Box
Unterstützung eines 48V AGM Akkussatzes

Bonus wäre ggf 1/2 schaltbare Ausgänge

Lg Robert

**Comment by Manos1966** [comment]
Ich wuerde einen anderen Victron empfehlen...
[Wiki](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Warning-about-Victron-MPPT-100-20)

**Reply by rs3hc** [reply]
Sag an... 😅

2-3x 420Wp

**Reply by kloppy1984** [reply]
Hey, ich habe eins auf Kleinanzeigen:

https://www.kleinanzeigen.de/s-anzeige/opendtu-onbattery-mit-pylontech-victron-mppt-shunt-huawei-/2565418448-168-9198

Wenn du bedarf hast, melde dich.

Beste Grüße

**Reply by rs3hc** [reply]
Klingt interessant

**Comment by alexseuf** [comment]
> Hey, ich habe eins auf Kleinanzeigen:
 
habe so eine ähnliche Platine entwickelt, Stromlaufplan und Layout siehe [hier:](https://www.akkudoktor.net/forum/migrated-forums-balkonsolar/opendtu-on-battery-platine-mit-maximal-ausbau-dc-dc-und-jk-bms-stecker/paged/2/#post-172350)
Wie ist denn deine Belegung der VE-direkt Stecker für Victron MPPT und Victron Smart Shunt:
verwendest du 1zu1 Kabel oder Original victron Kabel bei denen Pin2 un3 gekreutzt sind?
Für beides habe ich schon gute Argumente gehört:

**Reply by kloppy1984** [reply]
Ich verwende ein 1:1 Kabel, selbst gecrimmt mit 3x 0,34qmm (ein ganz normales "Ini-Kabel") GND / VCC / RX. 
Den TX lasse ich aus.  Auf Platinen Unterseite ist der ADUM1201 verbaut .
